fcrepo-http-commons/src/main/java/org/fcrepo/http/commons/exceptionhandlers/AccessControlExceptionMapper.java

```diff
@@ -36,15 +30,15 @@
  * @author gregjan
  */
 @Provider
-public class AccessControlExceptionMapper implements
-        ExceptionMapper<AccessControlException>, ExceptionDebugLogging {
+public class AccessControlExceptionMapper extends FedoraExceptionMapper<AccessControlException> {
 
-    private static final Logger LOGGER =
-        getLogger(AccessControlExceptionMapper.class);
+    @Override
+    protected ResponseBuilder status(final AccessControlException e) {
+        return super.status(Status.FORBIDDEN);
+    }
 
     @Override
-    public Response toResponse(final AccessControlException e) {
-        debugException(this, e, LOGGER);
-        return status(FORBIDDEN).build();
+    protected ResponseBuilder entity(final ResponseBuilder builder, final AccessControlException e) {
+        return builder.entity("This resource is forbidden.");
```

Comment:
The problem is that if the entity is not set,  then the content-type comes back as text/html.